### PR TITLE
Combine encoder networks

### LINKED ISSUE #19: encoder-combination
combine:
- txt encoder
- shape encoder

handles:
- triplets to tensor
- loss function
- pdate routine

### PR DESCRIPTION
**Pull request by maxim0815**

### CODE REVIEW

File: models/TripletEncoder.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        so far:
+            batch is list of class Triplet
+        '''
+        bs = len(batch)
+        max_length = 96
+        pos_desc_batch = torch.zeros((bs, max_length)).long()
+        neg_desc_batch = torch.zeros((bs, max_length)).long()
+        shape_batch = torch.zeros((bs, 32, 32, 32, 4))
+        for i, triplet in enumerate(batch):
+            pos_desc_batch[i] = torch.from_numpy(triplet.pos_desc).long()
```
Comment (kynesto) [inline models/TripletEncoder.py]: straight to device?

Comment (kynesto) [reply models/TripletEncoder.py]: dann sparen wir den if torch.cuda is availiable

**Review by kynesto — APPROVED** [review]
